src/Umbraco.Web/Editors/AuthenticationController.cs

```diff
@@ -141,5 +146,20 @@ public HttpResponseMessage PostLogout()
         {           
             return Request.CreateResponse(HttpStatusCode.OK);
         }
+
+        // From: http://stackoverflow.com/a/740431/5018
+        protected string GetIPAddress(HttpContextBase httpContext)

```

Comment:
Might be better to make this an extension method on HttpContextBase instead of having it protected here (which means really nothing else can use it). I'll pull this in and update that today.